provider/resources.go

```diff
@@ -107,6 +109,12 @@ func Provider() tfbridge.ProviderInfo {
 				},
 			},
 			"ns1_tsigkey": {Tok: makeResource(mainMod, "Tsigkey")},
+			"ns1_dnsview": {
+				Tok: makeResource(mainMod, "DnsView"),
```

Comment:
This should be `Dnsview`. We don't insert capitalization except where we remove `_`. See `tsigkey -> Tsigkey` above (it's not `TsigKey`). 

This should remove the breaking change in this PR>